docker_start.sh

```diff
@@ -4,4 +4,10 @@ npm i
 PORT=3000
 
 export NODE_PORT=${PORT}
-exec node dist/src/main/server.js -- ${PORT}
\ No newline at end of file
+
+for entry in /opt/*
```

Comment:
If not necessary, we can remove the logging

Reply:
I want to keep it for now because we might hit some issues in phoenix 1